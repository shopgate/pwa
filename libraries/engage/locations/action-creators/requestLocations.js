import { REQUEST_LOCATIONS } from '../constants';

/**
 * Dispatches the REQUEST_PRODUCT_LOCATIONS action.
 * @param {Object} params fetch params.
 * @return {Object} The REQUEST_PRODUCT_LOCATIONS action.
 */
const requestLocations = params => ({
  type: REQUEST_LOCATIONS,
  params,
});

export default requestLocations;
