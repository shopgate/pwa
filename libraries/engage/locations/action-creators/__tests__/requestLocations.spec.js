import requestLocations from '../requestLocations';

describe('engage > locations > actions-creators', () => {
  test('requestLocations', () => {
    expect(requestLocations({ codes: ['code1'] })).toMatchSnapshot();
  });
});
