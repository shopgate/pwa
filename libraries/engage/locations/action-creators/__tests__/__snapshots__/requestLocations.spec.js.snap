// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`engage > locations > actions-creators requestLocations 1`] = `
Object {
  "params": Object {
    "codes": Array [
      "code1",
    ],
  },
  "type": "REQUEST_LOCATIONS",
}
`;
